src/renderer/components/open-project-page/index.tsx

```diff
@@ -10,15 +10,32 @@ import { RecordingProjectContext, ThemeContext } from '../../contexts';
 import { Consumer, RecordingProject, SupportedTheme } from '../../types';
 import { ensureFolderExists, filename, openFilePicker } from '../../utils';
 import BackButton from '../back-button';
+import { Positional } from '../helper-components';
 
 import knownProjects from './known-projects';
 
 interface ProjectRowProps {
+  onRecordingButtonClick: MouseEventHandler<HTMLElement>;
+  onNext: Consumer<void>;
+  onBack: MouseEventHandler<HTMLElement>;
   project: RecordingProject | undefined;
   onClick: Consumer<RecordingProject>;
   selected: boolean;
 }
 
+let reservedStateValues = [];
```

Comment:
Why are you declaring them here?

Reply:
Because I need to use the list of reservedStates to check whether there are any recording set records there.

Reply:
And I have moved it into function scope.

Reply:
See my other comments for the updated code.